README.md

```diff
@@ -358,6 +366,7 @@ Quoting [Wikipedia](https://en.wikipedia.org/wiki/Live_coding)
 - [FARM](http://functional-art.org/) - Workshop on Functional Art, Music, Modeling, and Design.
 - [ICLI](http://www.liveinterfaces.org/) - International Conference on Live Interfaces.
 - [NIME](http://www.nime.org) - New Interfaces for Musical Expression conference.
+- [WAC](https://http://webaudioconf.com/) - The Web Audio Conference.
```

Comment:
looks like a typo here, extra http:

Reply:
Nice catch

Reply:
@a-1219, i'll wait for the fix on this. the rest looks good.

Reply:
Done! Thanks for noticing